eslint.config

```diff
@@ -0,0 +1,12 @@
+{
+  "parserOptions": { "ecmaVersion": 6 },
+  "extends": "airbnb-base",
+    rules: {
+    'no-shadow': 0,
+    'no-param-reassign': 0,
+    'eol-last': 0,
+    "indent": ["error", "tab", { "SwitchCase": 1 }],
+    'no-tabs': 0,
+    'skipBlankLines': true 
+  }
+}
```

Comment:
Consider using this file and it's content instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json